### PR TITLE
switch object to constructor

### PR DESCRIPTION
**Pull request by tkachenko1503**

### CODE REVIEW

File: sklad.js [hunk 1]
```diff
@@ -121,21 +118,42 @@ sklad.remove = function (key) {
 }
 
 /**
+ * @param {Object} e Объект события изменение значений в localStorage
```
Comment (vs-kurkin) [inline sklad.js]: переименуй в event

File: sklad.js [hunk 2]
```diff
@@ -121,21 +118,42 @@ sklad.remove = function (key) {
...
+/**
  * Проверяем наличие localStorage
  * если доступен возвращаем ссылку на него
  * если не доступен возвращаем false
  * @return {Storage|Boolean}
  */
 function checkAndGetStorage() {
-    var test = new Date;
+    var test = new Date();
     var stor;
```
Comment (vs-kurkin) [inline sklad.js]: var _storage;

File: sklad.js [hunk 3]
```diff
@@ -121,21 +118,42 @@ sklad.remove = function (key) {
...
+ */
+addListener('storage', sklad._triggerChangeEvent.bind(sklad), false);
+
+module.exports = sklad;
+
+/**
  * Проверяем наличие localStorage
  * если доступен возвращаем ссылку на него
  * если не доступен возвращаем false
  * @return {Storage|Boolean}
```
Comment (pavelrevers) [inline sklad.js:152]: {Storage|undefined}

File: sklad.js [hunk 4]
```diff
@@ -7,37 +7,24 @@
...
 
-var sklad = new EventEmitter();
+var sklad;
+var addListener = getListenerFunction();
 
-var storage = checkAndGetStorage();
-var addListener = getListenerFunc();
+function Sklad() {
+    EventEmitter.call(this);
+}
```
Comment (pavelrevers) [inline sklad.js]: Sklad.prototype.constructor = Sklad

**Pull request merged**